keiko-dynomite/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply from: "$rootDir/gradle/spek.gradle"
+
+dependencies {
+  compile project(":keiko-core")
+  compile("com.netflix.dyno:dyno-jedis:${spinnaker.version("dynomite")}") {
+    exclude group: 'joda-time'
+    exclude group: 'org.apache.httpcomponents'
+    exclude group: 'org.slf4j'
+  }
+  compile("com.netflix.dyno-queues:dyno-queues-redis:1.0.8") {
+    exclude group: 'com.netflix'
```

Comment:
Preferring our own transitive dependencies, rather than the ones dyno-queues defines (they're lagging behind).